### PR TITLE
Add new feature

### PR DESCRIPTION
**Pull request by newtonrd**
This is the shiz

### CODE REVIEW

File: scripts/example.coffee [hunk 1]
```diff
@@ -48,10 +48,10 @@ module.exports = (robot) ->
   #     return
   #   msg.send "#{answer}, but what is the question?"
   #
-  # robot.respond /you are a little slow/, (msg) ->
-  #   setTimeout () ->
-  #     msg.send "Who you calling 'slow'?"
-  #   , 60 * 1000
+    robot.respond /you are a little slow/, (msg) ->
+      setTimeout () ->
+        msg.send "Who you calling 'slow'?"
```
Comment (lydonb) [inline scripts/example.coffee:53]: Can you make this double quotes?

Comment (newtonrd) [inline scripts/example.coffee:53]: It already is :sheep:

Comment (newtonrd) [inline scripts/example.coffee:53]: :shipit:

**Comment by lydonb** [comment]
:shipit:

**Pull request merged**